services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -1147,6 +1148,12 @@ protected void initP2PNetwork() {
         new LocalOperationAcceptedFilter<>(p2pNetwork::publishSignedBlsToExecutionChange));
   }
 
+  protected void initEphemeryValidationService() {
+    final EphemerySlotValidationService slotValidationService =
+        new EphemerySlotValidationService(recentChainData);
+    eventChannels.subscribe(SlotEventsChannel.class, slotValidationService);
+  }
```

Comment:
can probably just initialise this if we're in ephemery, otherwise your slot processor would need to be capable of checking if its in ephemery...